tests/e2e/specs/rest-api/external-product.test.js

```diff
@@ -0,0 +1,6 @@
+/*
+ * Internal dependencies
+ */
+const { runExternalProductAPITest } = require( '@woocommerce/e2e-core-tests' );
```

Comment:
Once we start adding more of these, I wonder if we'll want having a separate task in to update the README with what these tests include and how to run them.

Reply:
Thanks!